packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -390,13 +450,7 @@ export default class Meeting extends StatelessSparkPlugin {
    * @memberof Meeting
    */
   leave() {
-    return this.meetingRequest.leaveMeeting({
-      locusUrl: this.locusUrl,
-      selfId: this.selfId,
-      correlationId: this.correlationId,
-      resourceId: null,
-      deviceUrl: this.deviceUrl
-    });
+    return MeetingUtil.leaveMeeting(this);
```

Comment:
I'd rather be explicit about what options we are passing here instead of passing the entire class. We can discuss at standup if you disagree.

Reply:
No I don't disagree, but we just need to get clear there and do it everywhere